moped-editor/src/utils/mapHelpers.js

```diff
@@ -252,6 +260,62 @@ export const mapConfig = {
         };
       },
     },
+    projectFeatures: {
+      layerLabel: "Project Features",
+      layerIdName: "projectFeatures",
+      layerIdField: "PROJECT_EXTENT_ID",
+      layerIdGetPath: "properties.PROJECT_EXTENT_ID",
+      layerOrder: 5,
+      layerColor: theme.palette.grey["800"],
+      layerMaxLOD: 12,
+      isClickEditable: false,
+      isInitiallyVisible: false,
+      get layerStyleSpec() {
+        return function() {
+          return {
+            id: this.layerIdName,
+            type: "line",
+            layout: {
+              "line-join": "round",
+              "line-cap": "round",
+            },
+            paint: {
+              "line-color": [
+                "case",
+                ["==", ["get", "LINE_TYPE"], TRAIL_LINE_TYPE],
+                theme.palette.grey["800"],
+                this.layerColor,
+              ],
+              "line-width": mapStyles.lineWidthStops,
+            },
+          };
+        };
+      },
+    },
+    projectFeaturePoints: {
+      layerDrawn: false,
+      layerLabel: "Project Points",
+      layerIdName: "projectFeaturePoints",
+      layerIdField: "PROJECT_EXTENT_ID",
+      layerIdGetPath: "properties.PROJECT_EXTENT_ID",
+      layerOrder: 6,
+      layerColor: theme.palette.grey["800"],
+      layerMaxLOD: 12,
+      isClickEditable: false,
+      isInitiallyVisible: false,
+      get layerStyleSpec() {
+        return function() {
+          return {
+            id: this.layerIdName,
+            type: "circle",
+            paint: {
+              "circle-color": this.layerColor,
+              "circle-radius": mapStyles.circleRadiusStops,
+            },
+          };
+        };
+      },
+    },
```

Comment:
These two large blocks implement the map style specifications for two additional layers:
- One to show the entire project's points in dark gray (when editing a new or existing component)
- Another to show the entire project line's in dark gray (when editing a new or existing component)